src/styles/blocks/main-block/about__us.scss

```
@@ -0,0 +1,109 @@
+.about {
+  &__us-title {
```

```suggestion
.about-us {
  &__title {
```
Probably, it would be better naming. Section called 'about us'